recipes/swig/all/test_package/conanfile.py

```diff
@@ -1,31 +1,17 @@
 from conans import CMake, ConanFile, tools
-from conans.errors import ConanException
-import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    generators = "cmake", "cmake_find_package"
 
-    # # FIXME: cpython is not available on CCI (yet)
-    # def build(self):
-    #     cmake = CMake(self)
-    #     cmake.configure(args=["--trace", "--trace-expand"])
-    #     cmake.build()
-    #
-    # def build_requirements(self):
-    #     self.build_requires("cpython/3.8.3")
-    #
-    # def requirements(self):
-    #     self.requires("cpython/3.8.3")
+    def build(self):
+        if self.settings.compiler != "Visual Studio":
```

Comment:
I think you disabled Visual Studio because it looked for debug python libraries?
Maybe add a `FIXME` that a python conan recipe can fix this.